src/libraries/oracle/OracleLibrary.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: UNLICENCED
```

Comment:
Here as well let's write where it comes from.

Also, let's just move all the libraries in `libraries/uniswap-v3-oracle` as these libraries are specific to the uniswap codebase.